interfaces/apparmor/apparmor_test.go

```diff
@@ -56,7 +56,7 @@ func (s *appArmorSuite) SetUpTest(c *C) {
 func (s *appArmorSuite) TestLoadProfileRunsAppArmorParserReplace(c *C) {
```

Comment:
My feeling is that we should update the tests to test a case with multi profiles and a test with no profile.